bin/plugin/commands/packages.js

```diff
@@ -105,13 +139,19 @@ async function runNpmReleaseBranchSyncStep( pluginReleaseBranch, config ) {
 			`>> Syncing the latest plugin release to "${ pluginReleaseBranch }".`
 		);
 
-		await git.replaceContentFromRemoteBranch(
-			gitWorkingDirectoryPath,
-			pluginReleaseBranch
-		);
+		const repo = git( gitWorkingDirectoryPath );
+
+		/*
+		 * Replace content from remote branch.
+		 *
+		 * @TODO: What is our goal here? Could `git reset --hard origin/${pluginReleaseBranch}` work?
+		 *        Why are we manually removing and then adding files back in?
```

Comment:
> What is our goal here? Could `git reset --hard origin/${pluginReleaseBranch}` work?

I was puzzled by the same question when I last touched this file. Let's give your suggestion a try :+1:

> Why are we manually removing and then adding files back in?

I think it's mostly to cover any files that were added and/or removed with respect to the other branch?

Reply:
@ockham do you think it would be fair to do this in a follow-up PR or make the change here? I'm nervous about smushing the logic change with the refactor in case it turns out that it was necessary to be the way it is.

Reply:
Makes sense. Follow-up is fine :+1: